cost/azure/rightsize_managed_disks/azure_rightsize_managed_disks.pt

```diff
@@ -1,13 +1,13 @@
-name "Azure Rightsize Managed Disks"
+name "Azure Rightsize Managed Disks v1.2"
```

Comment:
We usually do not add version to policy name. Should we consider to rename policy to be explicit that it downgrades disk type only.

Reply:
thank you! I put it accidentally! I just removed it.